src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    orange,
```

Comment:
enum values are constants so let's have proper naming
```suggestion
    RED,
    ORANGE,
```